### ISSUE TITLE
Voip consistency >= 100 players

### ISSUE THREAD

**Issue by RRavage** [Itokoyamato/TokoVOIP_TS3#29]
We've had some issues with toko voip intermittently cutting out for large groups of people when the server is >= 100 players. It generally happens to clusters of players in groups more than 2.

During this, there seems to a tokovoip resource warning that progressively increases in ms until the server drops back down to 99 slots.

I'm collecting stream clips now so you can see and hear it. I've also asked for more F8 logs from our players.

**Comment by RRavage** [comment]
> [     68312] Error resuming coroutine: @tokovoip_script/src/c_utils.lua:45: attempt to index a nil value (field '?')
> [     68328] stack traceback:
> [     68328]     @tokovoip_script/src/c_utils.lua:45: in function 'doRefreshAllPlayerData'
> [     68328]     citizen:/scripting/lua/scheduler.lua:169: in function <citizen:/scripting/lua/scheduler.lua:168>
> [     68344] Processing removal for player 113 (Carm)
> [     68344] reassigning ped: 0000000000000000 0
> [     68344] Error resuming coroutine: @tokovoip_script/src/c_utils.lua:45: attempt to index a nil value (field '?')
> [     68344] stack traceback:
> [     68359]     @tokovoip_script/src/c_utils.lua:45: in function 'doRefreshAllPlayerData'
> [     68359]     citizen:/scripting/lua/scheduler.lua:169: in function <citizen:/scripting/lua/scheduler.lua:168>
> [     68359] Processing removal for player 108 (LazerX7)
> [     68375] reassigning ped: 0000000000000000 0
> [     68375] Error resuming coroutine: @tokovoip_script/src/c_utils.lua:45: attempt to index a nil value (field '?')
> [     68391] stack traceback:
> [     68391]     @tokovoip_script/src/c_utils.lua:45: in function 'doRefreshAllPlayerData'
> [     68391]     citizen:/scripting/lua/scheduler.lua:169: in function <citizen:/scripting/lua/scheduler.lua:168>

**Comment by RRavage** [comment]
So I think the issue is related playersData returning nil at times at higher slots.

Change function at line 42 of **tokovoip_script/src/c_utils.lua**

```
function doRefreshAllPlayerData(serverData)
	for playerServerId, playerData in pairs(serverData) do
		for key, data in pairs(playerData) do
			if (not playersData[playerServerId]) then
				playersData[playerServerId] = {};
			end
			playersData[playerServerId][key] = {data = data, shared = true};
		end
	end

	for playerServerId, playerData in pairs(playersData) do
		for key, data in pairs(playerData) do
			if (not serverData[playerServerId]) then
				playersData[playerServerId] = nil;
			elseif (serverData[playerServerId][key] == nil) then
				playersData[playerServerId][key] = nil;
			end
		end
	end
end
```

**Comment by Itokoyamato** [comment]
Thanks for the issue. A similar fix has been applied in #25, but on the server side. Waiting for some testing to be done before merging to master.
